src/test/java/com/spotify/autoscaler/simulation/FakeBTCluster.java

```diff
@@ -115,9 +116,18 @@ public static BigtableClusterBuilder getClusterBuilderForFilePath(final Path pat
   // make it clear that we return a modified version of the cluster, like simulating a read from the
   // database
   public BigtableCluster getCluster() {
+    final int loadDelta = getMetricsForNow().loadDelta.intValue();
+    final int loadDeltaDiff = loadDelta - cluster.loadDelta();
+    final Optional<Integer> overridenMinNodes =
+        loadDelta == 0
+            ? Optional.empty()
```

Comment:
After the first inline conditional my brain stops :) could we rewrite this in some other way?

Reply:
@emmmile how about something like this? 
https://github.com/spotify/bigtable-autoscaler/pull/71
(i'm not sure which one i prefer and am open to other suggestions )